modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -891,6 +891,19 @@ definition {
 			}
 		}
 
+		if (isSet(inputExpression)) {
+			if ("${inputCondition}" == "true") {
+				ObjectAdmin.enableCondition();
+
+				Type(
+					locator1 = "ObjectAction#INPUT_EXPRESSION",
+					value1 = "${inputExpression}");
+			}
+			else if ("${inputCondition}" == "false") {
+				ObjectAdmin.disableCondition();
+			}
+		}
+
```

Comment:
I think I remember seeing this in another PR from you. If you need these changes for this test as well, you should close the original PR, commit on top of it, then resend the PR.